src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs_SkipDate.txt

```diff
@@ -0,0 +1,15 @@
+﻿
+// ******** FUNCTION VALUE AS PARAMETERS ********
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

The *Skip and override tests should be in the /interpreter override folder. 
They're interpreter specific issues; and the test harness will merge them together and apply the overrides.

Reply:
Looks strange (tests under one project, overrides in another) but I'll follow the existing pattern